### ISSUE TITLE
Add exception handling example

### ISSUE THREAD

**Issue by kevinburke** [twilio/twilio-php#391]
I'm trying to figure out how to handle "no phone numbers available" - what's the right path to import, what class of exception gets thrown - and neither the README, nor https://www.twilio.com/docs/libraries/php, nor any of the code snippets in the API docs has an example of an exception being caught.

It would be nice if there was a simple example of how to capture exceptions, especially since library users should expect exceptions to be thrown by the client and be prepared to defend against that.

**Comment by kevinburke** [comment]
I believe I added exception handling examples to the initial implementations of /docs/:language/install pages but I am not sure what happened to them.

**Comment by kevinburke** [comment]
(In particular I was confused about how to access the error code. I made two mistakes; one was trying to import `\Twilio\Rest\RestException` instead of `\Twilio\Exceptions\RestException`. I then tried to call `echo $e->code`, but this is a protected property. I had to read through the source code for twilio-php to ensure that I was getting the correct code, then [read the Exception docs](http://php.net/manual/en/class.exception.php) to figure out how to access the `code` property.)
